doc/user_guide/00-introduction.ipynb

```diff
@@ -21,7 +21,7 @@
     "## Table of Contents\n",
     "\n",
     "1. [What is Survival Analysis?](#What-is-Survival-Analysis?)\n",
-    "2. [The Veterans' Administration Lung Cancer Trial](#The-Veterans%27-Administration-Lung-Cancer-Trial)\n",
+    "2. [The Veterans' Administration Lung Cancer Trial](#The-Veterans%E2%80%99-Administration-Lung-Cancer-Trial)\n",
```

Comment:
CI now says `/home/runner/work/scikit-survival/scikit-survival/doc/user_guide/00-introduction.ipynb:35:undefined label: /user_guide/00-introduction.ipynb#the-veterans%e2%80%99-administration-lung-cancer-trial` but here's another(?) alternative based on https://scikit-survival.readthedocs.io/en/latest/user_guide/00-introduction.html

```suggestion
    "2. [The Veterans' Administration Lung Cancer Trial](#The-Veterans'-Administration-Lung-Cancer-Trial)\n",
```